registry.json

```diff
@@ -64,6 +64,14 @@
             "type": "EXPORTER",
             "bugsText": "From the Feedback Hub on Windows, use the *Apps > Paint 3D* categorise."
         },
+        {
+            "generator": "Mozilla Spoke *",
+            "name": "Mozilla Spoke",
+            "author": "Mozilla",
```

Comment:
nit: I would expect this to get displayed as something like `Mozilla Spoke, by Mozilla` in tools using this registry. Do you want to just name it "Spoke"?